### PR TITLE
Fix for alignment issues between clock and notifications

### PR DESCRIPTION
**Pull request by ryannair05**
Hope this helped. Thanks for your work on the animation.

### CODE REVIEW

**Comment by SyedMuhammadAhmad** [comment]
Yes the animation is working perfectly fine but i have one issue and its that padlock is showing in camera app when i swipe to the left to go in camera from lockscreen.

**Comment by ryannair05** [comment]
I'm not the developer; I would submit a new issue or ask SniperGER directly.

File: Tweak.xm [hunk 1]
```diff
@@ -136,7 +136,21 @@ static CGFloat offset = 0;
...
+%end
+
+%hook SBDashBoardAdjunctListView
+- (void)setFrame:(CGRect)frame {
+		frame = CGRectMake(0,frame.origin.y + 25,frame.size.width,frame.size.height);
+		%orig(frame);
+}
+
 
+%end
```
Comment (SniperGER) [inline Tweak.xm]:
```suggestion
%end	// %hook SBDashBoardAdjunctListView
```

File: Tweak.xm [hunk 2]
```diff
@@ -136,7 +136,21 @@ static CGFloat offset = 0;
 }
 %end	// %hook PKGlyphView
 
+%hook NCNotificationListCollectionView
+- (void)setFrame:(CGRect)frame {
+		frame = CGRectMake(frame.origin.x,frame.origin.y + 25,frame.size.width,frame.size.height);
+		%orig(frame);
+}
+%end
```
Comment (SniperGER) [inline Tweak.xm]:
```suggestion
%end 	// %hook NCNotificationListCollectionView
```

**Review by SniperGER — DISMISSED** [review]
Add "markers" (don't know what to call them) to visualize ending a %hook or %group.
This does not affect the tweak in any way, but keeps the code style consistent.

**Comment by ryannair05** [comment]
What do you want me to change from here? Github isn't working properly for this because it is outdated.

**Comment by SniperGER** [comment]
I think it's because I've requested two changes, but you've only commited one so far. I can't tell that for sure right now, because CodeHub on iPad doesn't show it as clear as GitHub does.

**Comment by ryannair05** [comment]
From what I see, both the changes were made; however, GitHub still shows the changes are still being requested and have not been pushed.

**Pull request merged**